WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -128,5 +134,18 @@ extension LocalNotification {
                 comment: "Message on the local notification to remind to continue the Blaze campaign creation."
             )
         }
+        enum ProductImageBackgroundUpload {
+            static let title = NSLocalizedString(
+                "localNotification.ProductImageUploader.title",
+                value: "Product Image Upload in Progress",
```

Comment:
For a more concise message, how about "Uploading in Progress"?

Reply:
I think if we change it to "uploading in progress," there will be a subject missing, like uploading of what? So, I'm changing it to "Image Upload in Progress." Please, let me know what you think.

Reply:
I'm not a native English speaker so I'm unsure as well 😅 Let's keep this for now then.